app/views/declaration/summary/item_section.scala.html

```diff
@@ -14,89 +14,289 @@
+                        content = Text(messages("site.change")),
+                        visuallyHiddenText = Some(messages("declaration.summary.items.item.cusCode.change", item.sequenceId))
+                    )
+                )
+            ))
+        )
+    ),
+    item.taricCodes.map(taricCodes =>
+        SummaryListRow(
+            classes = s"item-${item.sequenceId}-taricAdditionalCodes-row",
+            key = Key(
+                content = Text(messages("declaration.summary.items.item.taricAdditionalCodes"))
+            ),
+            value = Value(
+                content = Text(taricCodes.map(_.taricCode).mkString(", "))
+            ),
+            actions = Some(Actions(
+                items = Seq(
+                    ActionItem(
+                        href = controllers.declaration.routes.TaricCodeController.displayPage(mode, item.id).url,
+                        content = Text(messages("site.change")),
+                        visuallyHiddenText = Some(messages("declaration.summary.items.item.taricAdditionalCodes.change", item.sequenceId))
+                    )
+                )
+            ))
+        )
+    ),
+    item.nactCodes.map(nactCodes =>
+        SummaryListRow(
+            classes = s"item-${item.sequenceId}-nationalAdditionalCodes-row",
+            key = Key(
+                content = Text(messages("declaration.summary.items.item.nationalAdditionalCodes"))
+            ),
+            value = Value(
+                content = Text(nactCodes.map(_.nactCode).mkString(", "))
+            ),
+            actions = Some(Actions(
+                items = Seq(
+                    ActionItem(
+                        href = controllers.declaration.routes.NactCodeController.displayPage(mode, item.id).url,
+                        content = Text(messages("site.change")),
+                        visuallyHiddenText = Some(messages("declaration.summary.items.item.nationalAdditionalCodes.change", item.sequenceId))
+                    )
+                )
+            ))
+        )
+    ),
+    item.statisticalValue.map(statisticalValue =>
+        SummaryListRow(
+            classes = s"item-${item.sequenceId}-itemValue-row",
+            key = Key(
+                content = Text(messages("declaration.summary.items.item.itemValue"))
+            ),
+            value = Value(
+                content = Text(statisticalValue.statisticalValue)
+            ),
+            actions = Some(Actions(
+                items = Seq(
+                    ActionItem(
+                        href = controllers.declaration.routes.StatisticalValueController.displayPage(mode, item.id).url,
+                        content = Text(messages("site.change")),
+                        visuallyHiddenText = Some(messages("declaration.summary.items.item.itemValue.change", item.sequenceId))
+                    )
+                )
+            ))
+        )
+    ),
+    if(!journeyRequest.isType(DeclarationType.CLEARANCE))
```

Comment:
I think it'll be better to create a method `isNotATypeOf()` inside `JourneyRequest`.

Or use `isType` without negation and specify all journeys where this section should be presented

Reply:
That's how it was done before (in deleted commodity_measure view) - but I can take a look at this

Reply:
On the other hand this if works exactly as `filterNot` on Optional